File: app/src/main/java/ca/antonious/materialdaypicker/MainActivity.java
package ca.antonious.materialdaypicker;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Toast;

import java.util.List;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        MaterialDayPicker materialDayPicker = findViewById(R.id.dayPicker);
        materialDayPicker.setDaySelectionChangedListener(new MaterialDayPicker.DaySelectionChangedListener() {
            @Override
            public void onDaySelectionChanged(List<MaterialDayPicker.Weekday> selectedDays) {
                Toast.makeText(MainActivity.this, "Days Changed", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: app/src/main/java/ca/antonious/sampleapp/MainActivity.kt
package ca.antonious.sampleapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.open_java_sample_button
import kotlinx.android.synthetic.main.activity_main.open_kotlin_sample_button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        open_java_sample_button.setOnClickListener {
            startActivity(Intent(this@MainActivity, JavaSampleActivity::class.java))
        }

        open_kotlin_sample_button.setOnClickListener {
            startActivity(Intent(this@MainActivity, KotlinSampleActivity::class.java))
        }
    }
}
